client/src/components/SearchBar.js

```diff
@@ -0,0 +1,106 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+export default function SearchBar({
+  veggiesToRender,
+  fruitiesToRender,
+  saladsToRender,
+}) {
+  const [searchBarIsOpen, setSearchBarIsOpen] = useState(false);
+  const [inputField, setInputField] = useState("");
+  const [searchSolution, setSearchSolution] = useState([]);
+
+  const allToRender = [
+    ...veggiesToRender,
+    ...fruitiesToRender,
+    ...saladsToRender,
+  ];
+
+  function handleKey(event) {
+    if (event.key === "Enter") {
+      const solution = allToRender.filter((value) => {
+        return value.name
+          .toLowerCase()
+          .includes(event.target.value.toLowerCase());
+      });
+      console.log(solution, 111);
```

Comment:
I'm quite sure that you didn't want to simply log out the solution but do something else with it?